### PR TITLE
Remove swiftypesearch

### PR DESCRIPTION
**Pull request by martypowell**
- Removed swifttype specific code
- Added styles for google custom search

Addresses #237

### CODE REVIEW

File: dist/js/homepage.js [hunk 1]
```diff
@@ -5136,10 +5136,12 @@ baltimoreCounty.constants = (function constants() {
 	var baltCoGo = {
 		urls: {
 			api: {
-				geocodeServer: '//bcgis.baltimorecountymd.gov/arcgis/rest/services/Geocoders/CompositeGeocode_CS/GeocodeServer',
+				geocodeServer:
+                    '//bcgis.baltimorecountymd.gov/arcgis/rest/services/Geocoders/CompositeGeocode_CS/GeocodeServer',
```
Comment (jsmith176) [inline dist/js/homepage.js]: should this be indented?

File: dist/js/homepage.js [hunk 2]
```diff
@@ -5136,10 +5136,12 @@ baltimoreCounty.constants = (function constants() {
...
 				createReport: rootUrl + '/api/baltcogo/createreport',
 				getReport: rootUrl + '/api/citysourced/getreport',
-				getReportLatLng: rootUrl + '/api/citysourced/getreportsbylatlng',
+				getReportLatLng:
+                    rootUrl + '/api/citysourced/getreportsbylatlng',
```
Comment (jsmith176) [inline dist/js/homepage.js]: indent?

File: dist/js/homepage.js (hunk 1 above)
Comment (martypowell) [reply dist/js/homepage.js]: @jsmith176 notice this is in the dist folder, so we are not controlling the format

File: dist/js/homepage.js (hunk 2 above)
Comment (martypowell) [reply dist/js/homepage.js]: @jsmith176 notice this is in the dist folder, so we are not controlling the format

Comment (jsmith176) [reply dist/js/homepage.js]: That's right, I missed that. This is good to go.

**Pull request merged**